_data/gear_prices.yml

```diff
@@ -36,6 +36,9 @@ camping:
     - title: Bear Canister
       deposit: $75
       rental: $4
+    - title: Compass
+      deposit: $16
+      rental: $1
```

Comment:
Interesting -- we rent compasses now? I would imagine they're so easily lost and/or hard to keep track of. Neat, though. Seems like quality safety gear to have.

Reply:
We're testing out it as a pilot program to increase financial accessibility. There's been a lot of push back against forcing folks to buy a $16 compass that they probably will never learn how to actually use so this is an attempt to help with that. I'm hoping they don't all instantly disappear but its definitely possible >_<